eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -70,28 +70,29 @@ jobs:
           Write-Host "##vso[task.setvariable variable=BaseName;]$baseName"
         displayName: Generate $(BaseName)
 
-      - pwsh: >-
-          eng/New-TestResources.ps1
-          -BaseName $(BaseName)
-          -ServiceDirectory ${{ parameters.ServiceDirectory }}
-          -TestApplicationId $env:TEST_APPLICATION_ID
-          -TestApplicationSecret $env:TEST_APPLICATION_SECRET
-          -TestApplicationOid $env:TEST_APPLICATION_OID
-          -TenantId $env:TENANT_ID
-          -ProvisionerApplicationId $env:PROVISIONER_APPLICATION_ID
-          -ProvisionerApplicationSecret $env:PROVISIONER_APPLICATION_SECRET
-          -DeleteAfterHours 24
-          -Location westus2
-          -Force
-          -Verbose
-        env:
-          TEST_APPLICATION_ID: $(aad-azure-sdk-test-client-id)
-          TEST_APPLICATION_SECRET: $(aad-azure-sdk-test-client-secret)
-          TEST_APPLICATION_OID: $(aad-azure-sdk-test-client-oid)
-          TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-          PROVISIONER_APPLICATION_ID: $(provisioner-aad-id)
-          PROVISIONER_APPLICATION_SECRET: $(provisioner-aad-secret)
-        displayName: Provision Test Resources
+      - ${{ if ne(parameters.DisableAzureResourceCreation, 'true') }}:
```

Comment:
We should add `DisableAzureResourceCreation` to the parameters section (like a function signature) so it's obvious that you can use it. This is cosmetic, not blocking.